engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/SelectGameScreen.java

```diff
@@ -108,6 +129,29 @@ public String get() {
         });
 
         WidgetUtil.trySubscribe(this, "close", button -> triggerBackAnimation());
+
+    }
+
+    private void updateDescription(GameInfo item)
+    {
+        String mainWorldGenerator = item.getManifest()
+          .getWorldInfo(TerasologyConstants.MAIN_WORLD)
+          .getWorldGenerator()
+          .getObjectName()
+          .toString();
+
+        String modules = item.getManifest()
```

Comment:
I'd rename this to something like `commaSeparatedModules` so it's immediately clear what's going on with the stream calls :)